framework/main.py

```diff
@@ -21,9 +21,27 @@ def process_arguments(arguments):
     """
     parser = argparse.ArgumentParser(add_help=True)
     parser.add_argument("--conf", action="append", default=[])
+    parser.add_argument("--email")
 
     args, unknown = parser.parse_known_args(args=arguments)
+    for each_arg in unknown:
+        if each_arg.startswith("--html"):
+            if "=" in each_arg:
+                framework.config.ENV_DATA["html_path"] = each_arg.split("=", 1)[1]
+            else:
+                html_path_position = unknown.index("--html")
+                framework.config.ENV_DATA["html_path"] = unknown[html_path_position + 1]
+            break
+    # for cli_param in unknown:
+    #     if "--html" in cli_param:
+    #         pattern = r'--html(?:=|\s+)(/[^ ]+)'
+    #         match = re.search(pattern, cli_param)
+    #         if match:
+    #             framework.config.ENV_DATA["html"] = match.group(1)
+
```

Comment:
Isn't possible to parse the args directly by the argparse module, so add the parameter to "known" arguments and get its value via `args.html`?

Reply:
deleted

Reply:
I used your method for email